octoprint_mrbeam/iobeam/dust_manager.py

```diff
@@ -70,6 +75,10 @@ def __init__(self, plugin):
         self._fan_timers = []
         self._last_command = dict(action=None, value=None)
         self._just_initialized = False
+        self._fan_not_spinning_ts = None
+        self._fan_data_missing_ts = None
+        self._fan_data_missing_reported = None
+        self._fan_data_too_old_reported = None
```

Comment:
self._fan_not_spinning_reported is not initialized
```suggestion
        self._fan_not_spinning_ts = None
        self._fan_data_missing_ts = None
        self._fan_data_missing_reported = None
        self._fan_data_too_old_reported = None
        self._fan_not_spinning_reported= None
```

Reply:
ohh ups, good catch